src/com/facebook/buck/cli/BuildCommand.java

```diff
@@ -302,6 +302,7 @@ Build createBuild(
         console,
         buckConfig.getDefaultTestTimeoutMillis(),
         isCodeCoverageEnabled(),
+        buckConfig.getBooleanValue("test", "incl_no_location_classes", false),
```

Comment:
Correct.  For example, if this were true by default the classes generated by jmockit would also be included in code coverage.